icepyx/__init__.py

```diff
@@ -1,3 +1,17 @@
+from warnings import warn
+
+deprecation_msg = """icepyx v1.x is deprecated; the back-end systems on which it relies
```

Comment:
```suggestion
deprecation_msg = """icepyx v1.x is being deprecated; the back-end systems on which it relies
```

Adding the link was a great idea!